### PR TITLE
Share image

### PR DESCRIPTION
**Pull request by jo-mut**
Share image message

### CODE REVIEW

File: protocol/requests/share_image_message.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+var ErrShareMessageInvalidID = errors.New("share image message: invalid id")
+var ErrShareMessageEmptyUsers = errors.New("share image message: empty users")
+
+type ShareImageMessage struct {
+	MessageID   string           `json:"id"`
+	Users       []types.HexBytes `json:"users"`
+	ContentType int32            `json:"contentType"`
```
Comment (cammellos) [inline protocol/requests/share_image_message.go]: Do you need to pass `ContentType` from the client? Is it not always an image?

File: protocol/messenger_chats.go [hunk 2]
```diff
@@ -575,3 +575,45 @@ func (m *Messenger) clearHistory(id string) (*MessengerResponse, error) {
...
+	if err != nil {
+		return nil, err
+	}
+
+	var messages []*common.Message
+	for _, pk := range request.Users {
+		message := &common.Message{}
+		message.ChatId = pk.String()
+		message.Payload = msg.Payload
+		message.ContentType = protobuf.ChatMessage_ContentType(request.ContentType)
```
Comment (cammellos) [inline protocol/messenger_chats.go]: Do we not know the content type based on the fact that you are calling "ShareImageMessage"?

**Review by cammellos — COMMENTED** [review]
Left a few comments, it would also be good if you created tests verifying the behavior, it will help develop faster in status-go if you use the tests to verify it works. But you can send it to QA earlier in case they find some issues.

File: protocol/requests/share_image_message.go (hunk 1 above)
Comment (jo-mut) [reply protocol/requests/share_image_message.go]: yeah i actually don't have to

**Comment by jo-mut** [comment]
I will write the test and also remove any unnecessary code given its known its an image

**Review by Samyoul — APPROVED** [review]
Nice PR. Apart from the items that Andrea mentions, I don't see anything that needs attention.

**Comment by jo-mut** [comment]
> Nice PR. Apart from the items that Andrea mentions, I don't see anything that needs attention.

Thank you @Samyoul.

**Pull request merged**